src/styles/main.scss

```diff
@@ -0,0 +1,49 @@
+@import "./utils/mixins";
+
+.body {
+  margin: 0;
+}
+
+@keyframes rotation {
+  0% {
+    transform: rotate(0);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch {
+  position: relative;
+  margin: 200px auto;
+  width: 200px;
+  height: 200px;
+  border: 1px solid black;
+  border-radius: 10%;
+}
+
+.seconds-hand {
+  @include hand;
+
+  animation: rotation 60s steps(60, start) 0.5s infinite;
+}
+
+.minutes-hand {
+  @include hand;
+  background-color: blueviolet;
+  height: 20%;
+
+  animation: rotation 600s linear 0.5s infinite;
```

Comment:
why 600? how many seconds are there in a minute?

Reply:
`.seconds-hand` is responsible for 1 minute turn of the second's hand
`.minutes-hand` do 1 full turn of the minute's hand in 10 minutes